src/Examine/LuceneEngine/Providers/LuceneSearcher.cs

```diff
@@ -288,7 +297,19 @@ public void ScheduleReopen()
                             //if we're on NRT then we should just try to re-open
                             if (_luceneSearcher._nrtWriter != null)
                             {
-                                MaybeReopen();
+                                try
+                                {
+                                    MaybeReopen();
+
+                                }
+                                catch (Exception)
```

Comment:
what exception is actually thrown? that's the one that should be used.

Reply:
@Shazwazza I think I actually was planning to add logger there, forgot will do that soon :)

Reply:
@Shazwazza I will add logging there, but can't really remind myself what exception i hit here 🤦‍♂️